reviews/2021-k3s-annual.md

```diff
@@ -90,25 +63,26 @@ In the following, we provide a list of **ecosystem projects** that have collabor
 - automatic discovery of nvidia container runtime
 - containerd v1.5.x
 - traefik v2.x
-- selinux
-- 
+- selinux support
 
-**K3s community and adoption growth** in the last year:
+## Annual Review Contents
 
+### Include a link to your project’s devstats page.
+Here is a link to our devstats page: https://k3s.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m
+
+In reviewing this data, we noticed a few issues. We've opened the following issues to resolve them:
+ - [[bug] missing data for k3s](https://github.com/cncf/devstats/issues/320)
+ - [Change to the repos that are in k3s devstats](https://github.com/cncf/devstats/issues/319)
+
+Unfortunately, until these issues are resolved, it is difficult to draw accurate conclusions from the devstats page. In the meantime, here is some data that can be gleaned directly from GitHub:
 | Statistic | Sandbox | Current |
 |-|-|-|
 | [GitHub Stars](https://github.com/k3s-io/k3s/stargazers) | ~14k | ~18k |
 | [Commits](https://github.com/k3s-io/k3s/commits/master) | ~75/month | ~65/month |
 | [Merged PRs](https://github.com/k3s-io/k3s/pulls) | ~35/month | ~45/month |
 | [Releases](https://github.com/k3s-io/k3s/releases) | 100 | 171 |
 
-## Annual Review Contents
-
-### Include a link to your project’s devstats page. We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add colour to the numbers and graphs we will see on devstats.
-
-- The main K3s project averages [~730 contributions from ~120 contributors](https://k3s.devstats.cncf.io/d/74/contributions-chart?orgId=1&from=1598943600000&to=1633071599000&var-period=m&var-metric=contributions&var-repogroup_name=K3s) on average per month for the last year.
-
-### How many maintainers do you have, and which organizations are they from? (Feel free to link to an existing MAINTAINERS file if appropriate.)
+### How many maintainers do you have, and which organizations are they from?
```

Comment:
We should either keep all the parentheses comments in each top question, or get rid of all of them. Right now we have the maintainers without it, but the next question about adoption with it.

Reply:
i like removing it. can you delete it as a suggestion and ill commit that?